### PR TITLE
new css/register-login

### PR DESCRIPTION
**Pull request by nikunj482**

### CODE REVIEW

File: LR/glasses/models.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class person(models.Model):
     password = models.CharField(max_length=100, default="")
     confirmpassword = models.CharField(max_length=100, default="")
     gender = models.CharField(max_length=1, choices=GENDER_CHOICES, default="")
+    country = models.CharField(max_length=10,default="")
 
     def __str__(self):
```
Comment (sahildesai313) [inline LR/glasses/models.py:19]: end of code  please enter the space

File: LR/glasses/templates/otp.html [hunk 2]
```diff
@@ -10,8 +10,6 @@
   <body>
     <div class="otp-container">
       <h2>OTP Sent</h2>
```
Comment (sahildesai313) [inline LR/glasses/templates/otp.html:12]: remove space

File: LR/glasses/templates/otp.html [hunk 3]
```diff
@@ -21,23 +19,21 @@ <h2>OTP Sent</h2>
...
-      <hr />
+      <hr />  
       {% comment %} <p><strong>{{ email }}</strong>.</p> {% endcomment %}
       
+      <p><strong>{{ email }}</strong></p>
+      <form method="post">
 
-      <hr />
       <p><strong>{{ email }}</strong>.</p>
       <p>OTP: <strong>{{ fake_otp }}</strong></p>
```
Comment (sahildesai313) [inline LR/glasses/templates/otp.html:29]: remove space

File: LR/glasses/templates/otp.html [hunk 4]
```diff
@@ -21,23 +19,21 @@ <h2>OTP Sent</h2>
...
-
       <form method="post" >
         {% csrf_token %}
         <input type="hidden" name="email" value="{{ email }}" />
         <input type="hidden" name="fake_otp" value="{{ fake_otp }}" />
         <label for="entered_otp">Enter OTP</label>
-
         <input type="text" id="entered_otp" name="enter_otp" />
-
         <input type="text" id="entered_otp" name="entered_otp" />
```
Comment (sahildesai313) [inline LR/glasses/templates/otp.html:37]: be careful to  not write one line multiple time

**Review by sahildesai313 — CHANGES_REQUESTED** [review]

File: LR/glasses/models.py (hunk 1 above)
Comment (nikunj482) [reply LR/glasses/models.py:19]: done

File: LR/glasses/templates/otp.html (hunk 2 above)
Comment (nikunj482) [reply LR/glasses/templates/otp.html:12]: ok

File: LR/glasses/templates/otp.html (hunk 3 above)
Comment (nikunj482) [reply LR/glasses/templates/otp.html:29]: ok

File: LR/glasses/templates/otp.html (hunk 4 above)
Comment (nikunj482) [reply LR/glasses/templates/otp.html:37]: done

**Review by sahildesai313 — APPROVED** [review]

**Pull request merged**